lib/still/preprocessor.ex

```diff
@@ -109,12 +109,19 @@ defmodule Still.Preprocessor do
   Retrieves the preprocessor pipeline for the given file.
   """
   def for(%{input_file: file}) do
-    preprocessors()[Path.extname(file)]
+    preprocessors()
+    |> Enum.find(fn {key, _value} ->
+      if is_binary(key) do
+        Path.extname(file) == key
+      else
+        Regex.match?(key, file)
+      end
+    end)
```

Comment:
Please let's extract this to a helper function.

Reply:
👍